update.sh

```diff
@@ -186,6 +186,59 @@ EOF
 chmod +x /etc/update-motd.d/30ncp-dist-upgrade
 }
 
+# Update modsecurity config file only if user is already in buster and is used.
```

Comment:
it will be easier to do `is_active modsecurity && run_app modsecurity`, so the `cofigure()` section is run if the user has it activated already.

Also this should be done in the `updates` folder.

Reply:
Ok, I 'll refactor the code to be in the updates folder.